modules/gcp/jumpbox/main.tf

```diff
@@ -81,6 +81,20 @@ resource "google_compute_instance" "jumpbox" {
   }
 }
 
+resource "null_resource" "gcp_cleanup" {
```

Comment:
nit: maybe add some docs on why this is needed?

Reply:
added a comment, @gonzaloserrano